userspace/libsinsp/sinsp.h

```diff
@@ -954,7 +954,10 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 	void add_cri_socket_path(const std::string &path);
 	void set_cri_timeout(int64_t timeout_ms);
 	void set_cri_async(bool async);
+
+	// TODO DEPRECATED: drop this method after a release or two
 	void set_cri_delay(uint64_t delay_ms);
```

Comment:
What about using the `deprecated` gcc attribute too here?
https://gcc.gnu.org/onlinedocs/gcc-4.7.0/gcc/Function-Attributes.html (search for deprecated).

Reply:
I'm dropping it, since it looks like it breaks the Windows build.